app/assets/stylesheets/govuk_publishing_components/components/_contents-list.scss

```diff
@@ -53,13 +48,21 @@
     &:active {
       text-decoration: underline;
     }
+
+    &:focus {
+      @include govuk-focused-text;
```

Comment:
This all looks fine. If the focus state isn't quite right can I suggest creating a new mixin to use throughout the code in place of this one, something like:

```
@mixin link-focus-override {
  &:focus,
  &:active:focus,
  &:link:focus,
  &:visited:focus {
    @include govuk-focused-text;
    color: govuk-colour("black") !important;
    // plus anything else
  }
}
```

You may not need that exact combination of selectors, you'll need to try it out and see what works.